### DISCUSSION TITLE
parserError

### DISCUSSION THREAD

**Discussion by Vergudu** [Cyfrin/foundry-full-course-cu#160] [Q&A] [answered]
i can't compile my code. It gives me this parser Error

ParserError: Function, variable, struct or modifier declaration expected.
  --> SimpleStorage.sol:30:2:
   |
30 | }
   |  ^

**Comment by ghost** [comment]
Hello @Vergudu, is going to be difficult to help you with the little information you provided, can you format you question according to [this](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/how-to-ask-a-question.md)?

**Comment by cromewar** [comment, marked as answer]
Yes please @Vergudu use the formatting for questions mentioned by @pacelliv, in other hand I think that you must have a unclosed function or contract with `}`.
